### ISSUE TITLE
Output is non-deterministic

### ISSUE THREAD

**Issue by gshawn3** [EvilBT/ComfyUI_SLK_joy_caption_two#157]
This set of nodes is fantastic!! Thank you so much.

I noticed that the text output of the nodes is non-deterministic. It differs from one run to the next depending on what tasks have run on the GPU in the meantime. This is because there is no seed parameter.

I recommend setting a seed explicitly right before calling `text_model.generate` in the various `generate` methods to ensure that the output stays consistent from one run to the next. Alternatively, a seed widget could be exposed if desired, similar to other sampling nodes in ComfyUI.

```py
SEED = 12345  # Or any other integer
torch.manual_seed(SEED)
if torch.cuda.is_available():
    torch.cuda.manual_seed_all(SEED)

generate_ids = text_model.generate(input_ids, inputs_embeds=input_embeds, attention_mask=attention_mask,
                                   max_new_tokens=300, do_sample=True,
                                   suppress_tokens=None)  # Uses the default which is temp=0.6, top_p=0.9
```
